zerver/data_import/slack.py

```diff
@@ -314,7 +324,11 @@ def get_user_email(user: ZerverFieldsT, domain_name: str) -> str:
             slack_bot_name = user['profile']['first_name']
         else:
             raise AssertionError("Could not identify bot type")
-        return slack_bot_name.replace("Bot", "").replace(" ", "") + f"-bot@{domain_name}"
+        bot_email = slack_bot_name.replace("Bot", "").replace(" ", "") + f"-bot@{domain_name}"
```

Comment:
Why not use the value of "bot_id" which is unique to each bot for the email?

Reply:
With "bot_id" there is no need for `bot_id_count`.

Reply:
The same thought came to my mind .But @showell told me in https://chat.zulip.org/#narrow/stream/9-issues/topic/issue.20.2316793/near/1095942 that this approach is better.

Reply:
I think Steve's proposal makes sense to me; the advantage is that we don't have to make as many assumptions about Slack's data export system being free of bugs.